android/sdk/src/main/java/com/taobao/weex/ui/view/listview/adapter/ListBaseViewHolder.java

```diff
@@ -251,6 +252,10 @@ public void bindData(WXComponent component) {
       isRecycled = false;
     }
   }
+    
+  public boolean isFullSpan() {
```

Comment:
Since component can not change in holder, suggest move this detect to constructor method and keep it in a field.